content/news/2024-02-03-bevy-0.13/index.md

```diff
@@ -154,9 +154,170 @@ TODO.
 
 ## Extensionless asset support
 
-<div class="release-feature-authors">authors: @TODO</div>
+<div class="release-feature-authors">authors: @bushrat011899</div>
 
-TODO.
+### Some background
```

Comment:
I think this background section should be cut. And we just need a sentence that says that before the AssetLoader to use was inferred from the file extension.

Reply:
Perfectly reasonable, I decided to err on the side of verbosity since it's easier to cut in editing. Happy to remove that section as described.